.github/workflows/draft-release.yml

```diff
@@ -50,8 +50,9 @@ jobs:
       - name: Create draft release
         run: |
           RELEASE_NAME=syslog-ng-`cat VERSION`
+          RELEASE_ORIG_NAME=syslog-ng_`cat VERSION`
           sed "1 i${RELEASE_NAME}\n" NEWS.md > /tmp/message
-          mv dbld/build/${RELEASE_NAME}.orig.tar.gz dbld/build/${RELEASE_NAME}.tar.gz
+          mv dbld/build/${RELEASE_ORIG_NAME}.orig.tar.gz dbld/build/${RELEASE_NAME}.tar.gz
```

Comment:
We have another orig tar.gz in /build: `./dbld/build/syslog-ng-3.30.1.tar.gz.orig`
This one was saved before deb/rpm packages are built.
```
dbld/pkg-tarball:14:cp $SYSLOGNG_TARBALL ${SYSLOGNG_TARBALL}.orig
```

Reply:
> We have another orig tar.gz in /build: `./dbld/build/syslog-ng-3.30.1.tar.gz.orig`
> This one was saved before deb/rpm packages are built.
> 
> ```
> dbld/pkg-tarball:14:cp $SYSLOGNG_TARBALL ${SYSLOGNG_TARBALL}.orig
> ```

these are the same tar.gz files - but would be better to copy the tarball generated originally by the pkg-tarball...
I think this part of the dbld is not the best: release or tarball should generate the syslog-ng-$VERSION.tar.gz as real release tarball that can be uploaded.

Reply:
Yes they are the same, that's what I meant too.
I agree with you, `release` should generate a tarball that can be uploaded as a release tarball.